.github/workflows/ghstack_land.yaml

```diff
@@ -1,3 +1,25 @@
+# The land-command workflow runs ghstack land on the PR
+#
+# This workflow is triggered by leaving a "/land" comment on
```

Comment:
is it just `/land`? I was thinking it would be something like `@fmscibot /land`

Reply:
It is just `/land`, who performs the action is defined in the workflow setup. I think this provides a simpler interface to users. The actual command after the slash can be easily changed if you wanted to have `/<some command`. 
It should be possible to add parameters as well in future, if you wanted to have `/land <some-param>`.

Reply:
hmm... I'm afraid that text is simple enough show up in other contexts. Comments like "how much work is left before we can merge/land this PR?" could trigger the bot. Requiring the bot be tagged seems safer?

Reply:
The command must be at the beginning of the first line of the comment: https://github.com/peter-evans/slash-command-dispatch/blob/72fc7cd8a8c0804ad14e43723a40b310b2fbcbbf/src/main.ts#L58C4-L61

Reply:
ah OK, let's add that clarification to the comment

Reply:
Done.